stingray/modeling/parameterestimation.py

```diff
@@ -249,19 +271,25 @@ def fit(self, lpost, t0, neg=True, scipy_optimize_options=None):
         # at least until scipy 0.11 is out
         funcval = 100.0
         i = 0
+
         while funcval == 100 or funcval == 200 or \
                 funcval == 0.0 or not np.isfinite(funcval):
 
             if i > 20:
                 raise Exception("Fitting unsuccessful!")
             # perturb parameters slightly
-            t0_p = np.random.multivariate_normal(t0, np.diag(np.abs(t0)/100.))
+            t0_p = np.random.multivariate_normal(p0, np.diag(np.abs(p0)/100.))
```

Comment:
What, exactly, is the difference between `p0` and `t0`? I think having a comment line somewhere would be helpful to clear that up. :)